src/components/Chat/InputMessageV2.tsx

```diff
@@ -97,7 +116,7 @@ const styles = StyleSheet.create({
 });
 
 export interface InputMessageV2Props {
-  onSendButtonClicked: (message: string) => void;
+  onSendButtonClicked: (message: string, attachments: any) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onSendButtonClicked` function now takes an additional parameter `attachments`. This is a breaking change if there are other parts of the code that use this component and pass a function to this prop. Make sure all usages of this component are updated to pass a function that can handle this additional parameter.



<!-- This is an auto-generated comment by OSS CodeRabbit -->